src/main/java/com/example/onlinebookstore/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,73 @@
+package com.example.onlinebookstore.controller;
+
+import com.example.onlinebookstore.dto.cartitem.CreateCartItemRequestDto;
+import com.example.onlinebookstore.dto.cartitem.UpdateCartItemRequestDto;
+import com.example.onlinebookstore.dto.shoppingcart.ShoppingCartDto;
+import com.example.onlinebookstore.model.User;
+import com.example.onlinebookstore.service.ShoppingCartService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Shopping cart management", description = "Endpoints for managing shopping cart")
+@RestController
+@RequestMapping("/cart")
+@RequiredArgsConstructor
+public class ShoppingCartController {
```

Comment:
ShoppingCartController is not a part of this task

Reply:
not fixed

Reply:
not fixed